tests/unit/test_workflow_files.py

```diff
@@ -1523,3 +1524,24 @@ def test_get_rsync_rund_cmd(tmp_run_dir: Callable):
         '--exclude=log', '--exclude=work', '--exclude=share',
         '--exclude=_cylc-install', '--exclude=.service',
         'blah/', f'{cylc_run_dir}/']
+
+
+@pytest.mark.parametrize(
+    'expect, dirs',
+    [
+        (['run1'], ['run1', 'run2']),
+        (['run1', 'run11'], ['run1', 'run11', 'run2']),
+        (['run1200'], ['run1200', 'run1201']),
+        (['foo'], ['foo', 'bar']),
```

Comment:
`runN` isn't used for named (instead of numbered) runs

```suggestion
```

Reply:
No, I want to check that nothing untoward happens when one uses named runs.